### PR TITLE
add NGINX and remove duplicated label

### PR DESCRIPTION
**Pull request by yin1999**
### Description

add NGINX and remove duplicated label

### Motivation

We've added `nginx` label in [Yari](https://github.com/mdn/yari/blob/886b2c8090af69467d31caef77fb446831f6acdd/build/syntax-highlight.ts#L19-L63).
And the `diff` label is duplicated here.

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md [hunk 1]
```diff
@@ -94,14 +94,14 @@ On MDN, writers will use code fences for example code blocks. They must specify
   - `diff` - Diff file
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:94]:
```suggestion
```

File: files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md [hunk 2]
```diff
@@ -94,14 +94,14 @@ On MDN, writers will use code fences for example code blocks. They must specify
...
   - `ignore` - Gitignore file
   - `apacheconf` - Apache configuration
+  - `nginx` - NGINX configuration
 - Templates
   - `django` - Django templates
   - `svelte` - Svelte templates
   - `handlebars` - Handlebars templates
   - `pug` - [Pug templates](https://pugjs.org/api/getting-started.html) (which may be used by [Express](/en-US/docs/Learn/Server-side/Express_Nodejs/Displaying_data/Template_primer))
 - Other
   - `plain` - Plain text
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:104]:
```suggestion
  - `plain` - Plain text
  - `diff` - Diff file
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much @yin1999 . I chose to remove the other `diff` as it was the odd one out in the grouping. I made it "diff file" rather than "git diff" since presumably it doesn't have to be a diff from git.

**Comment by yin1999** [comment]
> Thanks very much @yin1999 . I chose to remove the other `diff` as it was the odd one out in the grouping. I made it "diff file" rather than "git diff" since presumably it doesn't have to be a diff from git.

Thanks for the correction 🤗

**Pull request merged**